tracking/src/DataCollection.js

```diff
@@ -0,0 +1,86 @@
+const NOT_AVAILABLE = "not available"
+class DataCollection {
+    constructor() {
+        console.log("DataCollection init"); 
+
+        this.getPlatform(); 
+        this.getLanguage(); 
+        this.getOS(); 
+        this.getScreenResolution(); 
+        this.getAppName(); 
+    }
+
+    toJson() {
+        const res = {}; 
+        res.p = this.platform; 
+        res.os = this.os; 
+        res.lang = this.language; 
+        res.sr = this.resolution; 
+        res.an = this.browser; 
```

Comment:
is this user agent? if so , no need to capture. It is automatic by server. "an" is meant for app name.

Reply:
never mind, just saw the code below. Optional to do this.